### DISCUSSION TITLE
Best way to backup log pipelines

### DISCUSSION THREAD

**Discussion by Ruppsn** [SigNoz/signoz#6109] [Q&A]
Hello, 

ist there a nice and easy way to backup all the pipelines in signoz?

Thanks in advance

**Comment by LOTRg** [comment]
If you are using SQLite which is the default for signoz, you can dump the table "pipelines" like so :
To get the current DB and it's location, find your "env" file which will be part of your systemd service as an EnvironmentFile, open that file and you should see a line like "#SIGNOZ_SQLSTORE_SQLITE_PATH", this is the path for signoz DB.
`sqlit3 <path_to_Signoz_DB> ` for example `sqlite3 /var/lib/signoz/signoz.db`
once inside the DB:
`.output <file_name_to_store.sql>
.dump pipelines
.output stdout`
This will get you a file in your current directory with the pipelines exported.
But Note:
Pipelines are tied to organization ID, so it's not just simply dump and import and it will work, you will have to change the  org_id field to match with your current org_id or whichever org_id you want to use these pipelines with.
To get your org_id, browse in sqlite3 the table named "organizations", the id column is the org_id you would need to update the pipelines org_id column to match.

If you are on Postgress, pgadmin4 is your friend to make make an export of the same table name.
